packages/core/src/utils/platform/wordJumpModifier.ts

```diff
@@ -2,3 +2,4 @@ import {isPlatform} from "./isPlatform";
 
 /** The modifier to make the arrow keys jump entire words at once */
 export const wordJumpModifier = isPlatform("mac") ? "alt" : "ctrl";
+export const wordDeleteModifier = isPlatform("mac") ? "alt" : "ctrl";
```

Comment:
I'm wondering whether we can just use the same `wordJumpModifier` here, would like some input from @sancarn too.

Reply:
I explicitly used a different name because the Jump modifier had such a specific name. Maybe another name something like `wordNavigationModifier` would be better since it allows both actions.

Reply:
Indeed can use the same word jump modifier, all word jumping behaviour is implemented with alt on mac, including selection, deletion etc.